examples/react-component/package.json

```diff
@@ -1,17 +1,29 @@
 {
   "name": "@examples/react-component",
   "private": true,
+  "exports": {
+    ".": {
+      "import": "./dist/esm/index.mjs",
+      "require": "./dist/cjs/index.js",
+      "types": "./dist/cjs/index.d.ts"
+    }
+  },
   "main": "./dist/cjs/index.js",
   "module": "./dist/esm/index.mjs",
   "types": "./dist/cjs/index.d.ts",
   "scripts": {
-    "build": "rslib build"
+    "build": "rslib build",
+    "serve": "pnpm build && npx http-server -p 3001 ./dist/ --cors"
```

Comment:
Can we use a folder named `module-federeation` with three projects:

- mf-host
- mf-remote
- react-component

it would be clearer for users to experience.

Reply:
At the same time, a `package.json` can be added with script `"mf:dev": "pnpm --filter mf-host dev & pnpm --filter mf-remote dev & pnpm --filter react-component run serve",` in this folder instead of the root package.json of whole monorepo.